README.md

```diff
@@ -2,7 +2,7 @@
 
 A 12 column responsive grid system for laying out documents, templates and modules.
 
-> Living off the grid and being kind of an outlaw brings a dangerous reality.  
+> Living off the grid and being kind of an outlaw brings a dangerous reality.
```

Comment:
No problem at all. `foo  bar` (2 spaces between two words) also works, in case that makes your editor happy.